public_suffix_list.dat

```diff
@@ -11338,6 +11338,10 @@ gotpantheon.com
 // Submitted by Steve Leung <steveleung@peplink.com>
 mypep.link
 
+// Publication Presse Communication SARL : https://ppcom.fr
```

Comment:
You need to move this down two entries (Pu > Pr) - Line 1348. So sorry this wasn't caught when you fixed the name.